app/(marketing)/TitleSection.tsx

```diff
@@ -0,0 +1,24 @@
+import Link from "next/link";
+
+const TitleSection = () => (
+  <div className="hero min-h-screen bg-base-100">
```

Comment:
ideas for a more dynamic background? maybe a primary→secondary gradient or something. If you'd rather just merge this in and then do the stylistic stuff if we have time, that's ok w/ me